packages/web-components/src/toggle-button/toggle-button.styles.ts

```diff
@@ -0,0 +1,107 @@
+import { css } from '@microsoft/fast-element';
+import { styles as ButtonStyles } from '../button/button.styles.js';
```

Comment:
Does anchor use these styles also? Should we have a generic location and a more generic name?

Reply:
It's technically the button styles and these are all subtle variations of button. I'm not sure a deeper extraction is needed here yet. I'd much prefer we start here and further abstract if we need to.